mollie_subscription_ept/controllers/main.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import logging
+import werkzeug
+from odoo import fields, http
+from odoo.http import request, Response
+from odoo.tools.json import scriptsafe as json_scriptsafe
+
+_logger = logging.getLogger(__name__)
+
+
+class WebsiteSale(http.Controller):
```

Comment:
It should inherit the WebsiteSale controller from website_sale module like it is done in /controllers/payment.py file.Also the function from /controllers/payment.py file can be added in this file instead of a separate file